masonry/src/event_loop_runner.rs

```diff
@@ -557,11 +557,12 @@ impl MasonryState<'_> {
                 }
             }
             WinitWindowEvent::MouseWheel { delta, .. } => {
-                // TODO - This delta value doesn't quite make sense.
-                // Figure out and document a better standard.
                 let delta = match delta {
+                    // 120 pixels is the default in Chrome and various other software
+                    // (when not overridden by configuration).
+                    // This is arbitrary, but better than nothing.
```

Comment:
Is this something we should be getting from the OS? That is, would this config be app specific?

I ask *only* to make this comment clearer (and maybe we should also link to an issue?)

Reply:
The ultimate solution is to allow the direct descendant of the portal to configure this at runtime, if true line deltas are desired, but for scrolling arbitrary content with unknown line/module sizes, 120 is a reasonable default.

Reply:
@DJMcNab I've opened #926 for further discussion on this matter.